protostar/cairo_testing/cairo1_test_collector.py

```diff
@@ -0,0 +1,75 @@
+from pathlib import Path
+from typing import List, Iterable
+
+import protostar.cairo.cairo_bindings as cairo1
+from protostar.testing import TestCollector
+from protostar.testing.test_collector import TestSuiteInfo
+from protostar.testing.test_suite import Cairo1TestSuite, TestSuite, TestCase
+
+
+class Cairo1TestCollectionException(Exception):
+    pass
+
+
+class Cairo1TestCollector(TestCollector):
```

Comment:
It's not hard but it strips our logic of skipping/selectively collecting tests. Otherwise i have to copy logic over. In my mind it's a temporary solution.

Reply:
![image](https://user-images.githubusercontent.com/34059895/221237244-915ddb74-f1cd-486c-bfd3-f2da5506b063.png)

Reply:
Discussed on slack, resolving